frontend/Checkout/components/Item/components/Product/components/Layout/components/ProductProperties/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Properties = ({ properties }) => (
+  <ul>
+    {properties.map(({ label, value }) => (
```

Comment:
Since the return value is not even used, that means `map` is just used for iterating over all elements. In that case a `forEach` loop would be more explicit. At first glance this looks like there might be a bug, even though it is meant to be that way.